File: projects/order-bc-query/core/src/main/java/com/robinfood/core/dtos/request/order/BrandDTO.java
package com.robinfood.core.dtos.request.order;

import lombok.AllArgsConstructor;
import lombok.Data;

import javax.validation.constraints.NotBlank;

@Data
@AllArgsConstructor
public class BrandDTO {

    private Long franchiseId;

    private Long id;

    @NotBlank
    private String name;
}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/dtos/transactionrequest/BrandDTO.kt
package com.robinfood.core.dtos.transactionrequest

import javax.validation.constraints.NotBlank
import javax.validation.constraints.NotNull
import javax.validation.constraints.Positive

data class BrandDTO (

    @NotNull
    @Positive
    val id: Long,

    @NotBlank
    val name: String
)